src/compound_targets.py

```diff
@@ -32,12 +37,18 @@ def cursorless_range(m) -> str:
     else:
         start = primitive_targets[0]
 
+    try:
+        is_column = m.cursorless_range_column is not None
+    except AttributeError:
+        is_column = False
+
     return {
         "type": "range",
         "start": start,
         "end": primitive_targets[-1],
         "excludeStart": not is_anchor_included(range_connective),
         "excludeEnd": not is_active_included(range_connective),
+        "isColumn": is_column,
```

Comment:
As above, tempted to generalize this one to `rangeType`, `modifier`, or `axis`, but maybe we should just keep it simple?

Reply:
I think arranged type could work